src/Symfony/Component/Messenger/Bridge/Doctrine/Tests/Transport/DoctrineIntegrationTest.php

```diff
@@ -56,6 +57,10 @@ public function testConnectionSendAndGet()
 
     public function testSendWithDelay()
     {
+        if (interface_exists(Result::class)) {
+            $this->markTestSkipped('Skip on Doctrine DBAL 2');
+        }
```

Comment:
This does not feel right and looks like something I might have broken. Let me check that.

Reply:
#51624

Reply:
Please revert this change. The test should pass now with DBAL 2 as well.